spec/support/schemas/claims_api/claims.json

```diff
@@ -12,10 +12,12 @@
         "properties": {
           "id": {
             "type": "string",
-            "description": "Claim ID from EVSS"
+            "description": "Claim ID from EVSS",
+            "example": "8347210"
```

Comment:
same here, why are all these example values suddenly here?

Reply:
Because I explicitly added them here.
I did a side-by-side comparison of what the v2 docs look like today in dev, vs. what I was developing.
What's deployed today (`swagger_blocks`) provides example values in the schema definitions.
So I went and added the same example values to the schemas so that `rswag` would generate docs that included the same example values.